<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PassengerDialog</class>
 <widget class="QDialog" name="PassengerDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>372</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Пассажир</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>428</width>
     <height>277</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="num_passport_label">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Номер паспорта:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="place_passport_label">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Место выдачи паспорта:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="date_passport_label">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Дата выдачи паспорта:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="surname_label">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Фамилия:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="name_label">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Имя:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="patronym_label">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Отчество:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="birthday_label">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Дата рождения:</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLineEdit" name="num_passport_edit"/>
      </item>
      <item>
       <widget class="QLineEdit" name="place_passport_edit"/>
      </item>
      <item>
       <widget class="QDateEdit" name="date_passport_date"/>
      </item>
      <item>
       <widget class="QLineEdit" name="surname_edit"/>
      </item>
      <item>
       <widget class="QLineEdit" name="name_edit"/>
      </item>
      <item>
       <widget class="QLineEdit" name="patronym_edit"/>
      </item>
      <item>
       <widget class="QDateEdit" name="birthday_date"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>330</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="title_label">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>10</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Введите данные пассажира</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
